### DISCUSSION TITLE
Blazor Mobile Bindings

### DISCUSSION THREAD

**Discussion by justinasmussen** [dotnet/maui#946] [General]
Are the Blazor Mobile Bindings part of MAUI and will they be in .NET 6?

**Comment by Ghevi** [comment]
Would like to know this aswell, do I need to create a separated project and then target it from the WebView component? Or is it already in a MAUI Blazor project by default?

**Comment by saint4eva** [comment]
> Are the Blazor Mobile Bindings part of MAUI and will they be in .NET 6?

No.
